images/miq-app/Dockerfile

```diff
@@ -53,8 +49,7 @@ RUN yum -y install centos-release-scl-rh \
                    openscap-scanner        \
                    patch                   \
                    psmisc                  \
-                   rh-postgresql95-postgresql-devel  \
-                   rh-postgresql95-postgresql-libs \
+                   postgresql-devel        \
```

Comment:
Is this versioned? Where are we getting this from?

Reply:
Our copr repo: https://copr.fedorainfracloud.org/coprs/manageiq/ManageIQ-Master/packages/

Reply:
Right, just found that ... do we want to rebuild it at some point? We're a few versions behind.

Oh, also we probably need to create an Ivanchuk repo, but this is fine we can merge this.

Reply:
Ivanchuk repo is already created. Didn't know we're already behind on version!!!  Will take a look.

Reply:
Yeah, think we're at 10.6 and 10.9 was released in June.